### PR TITLE
fix(accordion): update accordion breakpoints

### LINKED ISSUE #4993: Accordion content has padding right 25% applies
## What package(s) are you using?

- [x] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.
Accordion content has 25% padding, this is taking a lot of space and didn't used to be like this in v9.

Screen Shot 2020-01-09 at 2 48 01 PM

> Is this issue related to a specific component?
AccordionItem

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?
Chrome Version 79.0.3945.88 (Official Build) (64-bit)

> What version of the Carbon Design System are you using?
carbon-components-react: 7.9.0
carbon-components: 10.9.0

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

IBM Cloud

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

### PR DESCRIPTION
**Pull request by tw15egan**
Closes https://github.com/carbon-design-system/carbon/issues/4993

Updates Accordion to only use `16px` of padding when the screen is below `480px` Based on updated specs [found here](https://github.com/carbon-design-system/carbon/issues/4993#issuecomment-573199723) 

image

image

image

#### Changelog

**New**

- Breakpoints at `480px` and `640px`

#### Testing / Reviewing

See that the Accordion content shifts from `1rem` padding, to `3rem` padding, to `25%` padding as the screen size increases.

### CODE REVIEW

**Review by joshblack — APPROVED** [review]

File: packages/components/src/components/accordion/_accordion.scss [hunk 1]
```diff
@@ -98,12 +98,17 @@
     // Transition property for when the accordion closes
     transition: padding motion(standard, productive) $duration--fast-02;
     padding-left: $carbon--spacing-05;
-    padding-right: 25%;
+    padding-right: $carbon--spacing-05;
 
-    @include carbon--breakpoint-down('md') {
+    // Custom breakpoints based on issue #4993
+    @media (min-width: 480px) {
```
Comment (joshblack) [inline packages/components/src/components/accordion/_accordion.scss]: I think you could use `carbon--breakdown-up` if you want! e.g.

```scss
@include carbon--breakpoint-up(480px) {
  // ...
}
```

But it's basically the same thing 😂

Comment (tw15egan) [reply packages/components/src/components/accordion/_accordion.scss]: Oh nice, I wasn't sure if we had one for constructing custom breakpoints (hadn't seen `-down` or `-up`). I'll update it for consistency.

**Review by aagonzales — APPROVED** [review]
Looks good!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @tw15egan!

**Pull request merged**